packages/dds/tree/src/test/memory/.mocharc.cjs

```diff
@@ -4,16 +4,23 @@
  */
 
 /**
- * Mocha configuration file to run memory-profiling tests
+ * Mocha configuration file for memory profiling tests
  */
 
-module.exports = {
+const getFluidTreeTestMochaConfig = require("../../../.mocharc.cjs");
+
+const extendedConfig = {
+	...getFluidTreeTestMochaConfig,
 	"fgrep": ["@Benchmark", "@MemoryUsage"],
-	"node-option": ["expose-gc", "gc-global", "unhandled-rejections=strict"], // without leading "--"
-	"recursive": true,
-	"reporter": "@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js",
+	"node-option": [
+		"conditions=allow-ff-test-exports",
+		"expose-gc",
+		"gc-global",
+		"unhandled-rejections=strict",
+	], // without leading "--"
+	"reporter": "@fluid-tools/benchmark/dist/MochaMemoryTestReporter.js", // Changed reporter option to use the memory test reporter.
 	"reporterOptions": ["reportDir=.memoryTestsOutput/"],
-	"require": ["node_modules/@fluid-internal/mocha-test-setup"],
-	"spec": ["lib/test/memory/**/*.spec.*js", "--perfMode"],
-	"timeout": "90000",
+	"timeout": "360000", // depending on the test and the size of the E2E document, the timeout might not be enough. To address it, let's first try to decrease the number of iterations (minSampleCount).
```

Comment:
These comments came from another mocha rc file, right? I think we can remove them here, and set the timeout back to 90s if possible.

Reply:
Thanks for the quick review 😄 . 

Yes. The comment on `timeout` is from other mocha config. Switched back to `90000`.

Reply:
In addition, I don't think we need to override the `spec` property here, since the extended tree test mocha config configures `test/**`.